lib/rubocop/cop/netlify/require_scope_semantics.rb

```diff
@@ -16,22 +16,26 @@ module Netlify
       #   require_scope "all:read"
       #   def index
       class RequireScopeSemantics < RequireScopeBase
+        WRITE_KEYWORDS = ["update", "create", "destroy", "new", "revoke", "delete"].freeze
+        READ_KEYWORDS = ["show", "index", "edit"].freeze
```

Comment:
I feel like `new` and `edit` should go in the same category, either both in `WRITE` or both in `READ`. While typically these don't do the actual action of saving to the DB, it might be worth putting them in `WRITE`?

Reply:
OMG. Great catch!